java/ql/src/semmle/code/java/security/SensitiveActions.qll

```diff
@@ -27,6 +27,14 @@ private string nonSuspicious() {
   result = "%crypt%"
 }
 
+/**
+ * Gets a regular expression for matching common names of variables that indicate the value being held contains sensitive information.
+ */
+string getCommonSensitiveInfoRegex() {
```

Comment:
Now make `SensitiveInfoLog.ql` use this (making sure not to lose cases from its `getACredentialRegex` routine)

Reply:
Updated <code>SensitiveInfoLog.ql</code>